__tests__/api.spec.js

```diff
@@ -1,4 +1,5 @@
-import { API } from '../api';
+import "babel-polyfill";
```

Comment:
That's ok for this iteration but if several tests need `babel-polyfill`, we could use common jest setup